lms/templates/learner_dashboard/program_details_fragment.html

```diff
@@ -18,7 +18,7 @@
     certificateData: ${certificate_data | n, dump_js_escaped_json},
     urls: ${urls | n, dump_js_escaped_json},
     userPreferences: ${user_preferences | n, dump_js_escaped_json},
-    creditPathways: ${credit_pathways | n, dump_js_escaped_json},
+    creditPathways: ${pathways | n, dump_js_escaped_json},
```

Comment:
Should we change the creditPathways in the  program_details_fragment here, or wait until we officially split up pathways into credit and professional?

Reply:
I was thinking that the renaming could happen when we do the split. This already touches a lot of files in a lot of different places, and I wanted to minimize the footprint as much as possible.

Reply:
👍